app/javascript/packs/application.js

```diff
@@ -9,7 +9,8 @@ require("@rails/activestorage").start()
 require("channels")
 import 'controllers'
 
-// import 'bootstrap'
+import 'popper.js'
+import 'bootstrap'
```

Comment:
the flash messages are handled readily with bootstrap.  Bootstrap 5 depends on popper.